### ISSUE TITLE
Jira password stored not as a plain text

### ISSUE THREAD

**Issue by marek-pietrzak-tg** [danielkmariam/jiran#6]
Jira password is stored in a plain text in `config.json` file. 
I guess using some kind of hashing method/token is on your todo list @danielkmariam ?

**Comment by danielkmariam** [comment]
Correct, I am checking if token is possible otherwise will do the hash.

**Comment by danielkmariam** [comment]
@mheki yes it is on my to do, ideally with token
